src/parser/tptp/Tptp.g

```diff
@@ -258,7 +258,7 @@ parseCommand returns [cvc5::Command* cmd = NULL]
       }
       seq->addCommand(new SetInfoCommand("filename", filename));
       if(PARSER_STATE->hasConjecture()) {
-        seq->addCommand(new QueryCommand(SOLVER->mkFalse()));
+        seq->addCommand(new CheckSatCommand());
       } else {
         seq->addCommand(new CheckSatCommand());
```

Comment:
It looks like we can unify the branches.

Reply:
Actually I'm going to change Query(false) -> CheckSatAssuming(true) here, to maintain the behavior.   The may allow us to be more precise wrt TPTP status in the future if we want.